filters/auth/authclient.go

```diff
@@ -43,20 +43,20 @@ func newAuthClient(baseURL, spanName string, timeout time.Duration, maxIdleConns
 		return nil, err
 	}
 
-	tr := net.NewTransport(net.Options{
-		ResponseHeaderTimeout: timeout,
-		TLSHandshakeTimeout:   timeout,
-		MaxIdleConnsPerHost:   maxIdleConns,
-		Tracer:                tracer,
+	cli := net.NewClient(net.Options{
```

Comment:
we either should set the logger here, or use a different default in net.NewClient, because otherwise we will mix together access and application logs

Reply:
changed in NewClient